pkgs/tools/misc/xvfb-run/default.nix

```diff
@@ -29,9 +29,9 @@ stdenv.mkDerivation rec {
   '';
 
   meta = with stdenv.lib; {
-    description = ''Starter script for an instance of Xvfb, the "fake" X server'';
-    inherit (src.meta) homepage;
-    license = licenses.gpl2;
+    description = "A wrapper for the Xvfb command which simplifies the task of running commands";
+    homepage = src.meta.homepage +  "debian/local";
```

Comment:
It's good to be able to just read the URL there...

Reply:
I saw the usage of `inherit (src.meta) homepage;` in some packages lately and I liked the style.

In this case, the homepage line gets noticeably shorter, from 107 (when using the full path to `debian/local`) to 74 characters.

```
    homepage = "https://salsa.debian.org/xorg-team/xserver/xorg-server/tree/debian-unstable/debian/local";
```

vs

```
    homepage = src.meta.homepage +  "/tree/debian-unstable/debian/local";
```
(current version is incomplete, will be fixed)

Reply:
Yeah. Personally, I am not a fan of `src.meta.homepage` either. I frequently click on `meta.homepage` URL when reviewing PRs and this prevents this use case without evaluation the expression.